development/webpack/test/webpack.config.test.ts

```diff
@@ -162,6 +162,7 @@ ${Object.entries(env)
       {
         manifest_version: 3,
         name: 'name',
+        permissions: ['tabs'],
```

Comment:
Note: This is because now we have enabled test in args, then the previous logic will add this function addTabsPermission
```
if (args.test) {
    // test builds need "tabs" permission for switchToWindowWithTitle
    transforms.push(addTabsPermission);
  }
```